### DISCUSSION TITLE
No bundles were parsed.

### DISCUSSION THREAD

**Discussion by jrubner** [webpack-contrib/webpack-bundle-analyzer#633] [Q&A] [answered]
Hello,

and yes, i did search for those topics but i simply just couldn`t understand it or it was not the exact same problem.

Angular 18

I added this scripts to my package.json:

` "build:stats": "ng build --stats-json --configuration production", `
` "analyze": "webpack-bundle-analyzer dist/test/stats.json", `

The first one did indeed create the json file and the second one startet the analyzer.

But i get this Message, and no information is given in the Analyzer:

> webpack-bundle-analyzer dist/test/stats.json
No bundles were parsed. Analyzer will show only original module sizes from stats file.

![image](https://github.com/webpack-contrib/webpack-bundle-analyzer/assets/64382752/2b32e296-f383-492f-be2a-1ad02042527c)

i tried this for half a day, also tried adding it as a plugin but couldn`t make it work.

Thanks in advance.

**Comment by valscion** [comment, marked as answer]
> dist/test/stats.json

Is `dist/test` the directory where `ng build --stats-json --configuration production` outputs your resulting bundle files or might the resulting bundle be in some other directory, such as `dist/production/` or something like that?
